src/ucp/wireup/select.c

```diff
@@ -592,7 +592,7 @@ static UCS_F_NOINLINE ucs_status_t ucp_wireup_select_transport(
         }
 
         is_reachable = 0;
-
+        ucs_snprintf_safe(info_str, info_str_size, "unreachable");
```

Comment:
maybe use snprintf here for consistency then? In this function only `snprintf` is used

Reply:
fixed